ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -1,3 +1,26 @@
+- name: (linux) Install CrowdStrike Falcon
+  hosts: linux
+  become: true
+  gather_facts: yes
+  vars:
+    falcon_client_id: "{{ lookup('env', 'CROWDSTRIKE_CLIENT_ID') }}"
+    falcon_client_secret: "{{ lookup('env', 'CROWDSTRIKE_CLIENT_SECRET') }}"
+    falcon_customer_id: "{{ lookup('env', 'CROWDSTRIKE_CUSTOMER_ID') }}"
+    api_base_url: "https://api.laggar.gcw.crowdstrike.com"
+  roles:
+    - caos.ansible_roles.install_crowdstrike_falcon
+
+- name: (windows) Install CrowdStrike Falcon
+  hosts: windows
+  gather_facts: yes
+  vars:
+    falcon_client_id: "{{ lookup('env', 'CROWDSTRIKE_CLIENT_ID') }}"
+    falcon_client_secret: "{{ lookup('env', 'CROWDSTRIKE_CLIENT_SECRET') }}"
+    falcon_customer_id: "{{ lookup('env', 'CROWDSTRIKE_CUSTOMER_ID') }}"
+    api_base_url: "https://api.laggar.gcw.crowdstrike.com"
+    win_download_path: 'C:\Windows\Temp\falcon-sensor.exe'
+  roles:
+    - caos.ansible_roles.install_crowdstrike_falcon
```

Comment:
Wouldn't it just work if we specified this block with `hosts: all` instead? My understanding is that the Linux hosts would just ignore the `win_download_path` variable and that's all, right? Ah, or does it have to do with the use of `become: true`? If this is the case, then please leave a comment here explaining why we need to do it in separate blocks so we don't accidentally remove this block in the future because we forget about it.

Reply:
>Ah, or does it have to do with the use of become: true? 
precisely